dnsapi/dns_freedns.sh

```diff
@@ -138,76 +96,44 @@ dns_freedns_rm() {
   FREEDNS_COOKIE="$(_read_conf "$ACCOUNT_CONF_PATH" "FREEDNS_COOKIE")"
```

Comment:
why you need to save the cookie ?
And never use `_read_conf()` in your code.

Reply:
Once you login to FreeDNS for the first time you are issued a cookie to use for all future requests.  The cookie becomes invalid if user changes their password.  You don't want to save the userid/password and login every time, instead you use the cookie on every request.  This diff/compare also clips of the comments inline in the code that explained this. namely...
```
# Need to read cookie from conf file again in case new value set
# during login to FreeDNS when TXT record was created.
# acme.sh does not have a _readaccountconf() function
FREEDNS_COOKIE="$(_read_conf "$ACCOUNT_CONF_PATH" "FREEDNS_COOKIE")"
```
Of course as I look at acme.sh now, I see that you do have a readaccountconf() function that did not exist when I originally wrote this.  So, I can change to use that.  I also see a "mutable" version... can you explain those please?
Thanks

Reply:
Made the change to use _readaccountconf()